### PR TITLE
Migrate to politeia tlog API

### PR DESCRIPTION
**Pull request by crux25**
This PR implements the changes needed to migrate to the new Politeia tlog API. 
Specifically this PR:

1. Drops the use of go-piparser module.
2. Fetch chartdata from politeia API and save it with proposal object in stormdb.
3. Drop use of Axios in proposal module and instead use a more lightweight helper function.
4. Removes piclient and stop saving proposal related information to PgDB.
5. Drop use of refid in request contexts.
6. Clean up codes.

### CODE REVIEW

**Review by Ekeh — CHANGES_REQUESTED** [review]
Kindly, check the display of the proposal on mobile and adjust the margin from the top.

proposal_page.

**Comment by crux25** [comment]
> Kindly, check the display of the proposal on mobile and adjust the margin from the top.
> 
> proposal_page.
This is fixed now.

**Review by Ekeh — APPROVED** [review]
This PR has been reviewed and all looks good.

File: gov/politeia/handler.go [hunk 1]
```diff
@@ -184,19 +217,19 @@ func (prop *proposals) getProposalChartData(w http.ResponseWriter, r *http.Reque
...
 	return hash
 }
 
-// proposalPathCtx embeds "proposalrefID" into the request context
+// proposalPathCtx embeds "proposaltoken" into the request context
 func proposalPathCtx(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		proposalRefID := chi.URLParam(r, "proposalrefid")
-		ctx := context.WithValue(r.Context(), web.CtxProposalRefID, proposalRefID)
+		proposalToken := chi.URLParam(r, "proposaltoke")
```
[the remark this answers is not in the document]
Comment (crux25) [reply gov/politeia/handler.go]: I have fixed it in c5e5bd59cbfbb8231071ec71470a805c5bf84c51

**Pull request merged**